src/js/browser-plugin.js

```diff
@@ -314,14 +307,15 @@ const publishTweetAlertMisinfo = (label, url, tweetText) => {
   let popupTitle = browserAPI.i18n.getMessage('content_tagged_as', auxlabel);
   let popupButtonText = browserAPI.i18n.getMessage('ok');
   
+  log2Server('publish tweet', url, `Tweet id: ${tweetText}`, 'detected "not credible" tweet before publish'); 
```

Comment:
Please use the function argument "label" on the log action message "detected $auxlabel tweet...". For in case the misinformation label name is changed in the future.